docs/guide/data/idraw-readme.txt

```diff
@@ -0,0 +1,459 @@
+     drawings made up of graphics like text, lines, splines, rec-
+     tangles, polygons, and ellipses.   Drawings  are  stored  in
+     files  that can be printed on a PostScript printer.  You can
+     can open an existing drawing when starting up idraw by  typ-
+     ing a file name on the command line.
+
+     Idraw displays a portrait or landscape view of an 8.5 by  11
+     inch  page in its drawing area.  In a column along the draw-
+     ing area's left side is a set of  drawing  tool  icons,  and
+     above  the drawing area is a set of pull-down menus contain-
+     ing commands.  A panner in the lower left  corner  lets  you
+     pan  and  zoom the the drawing area.  Along the top is a set
+     of indicators that display editing information.
+
+DRAWING TOOLS
+     You must engage a tool before you can use it.  You engage  a
+     tool  by  clicking  on  its  icon or by typing the character
+     below and to the right of its icon.  The icon of the drawing
+     tool  that's  engaged  appears  in  inverted  colors.   Once
+     engaged, you use the tool by clicking the left mouse  button
+     in the drawing area.
+
+     The Select, Move, Scale, Stretch, Rotate,  and  Alter  tools
+     manipulate  existing  graphics.  Magnify makes a part of the
+     view expand to fill the entire view.  Text, Line, Multiline,
+     Open  Spline, Ellipse, Rectangle, Polygon, and Closed Spline
+     create new graphics.  Each tool works as follows:
+
+     Select         Select a graphic, unselecting all others.   A
+                    graphic  is selected if its handles are visi-
+                    ble.  Handles are small inverse-video squares
+                    that either surround the graphic or demarcate
+                    its important points (such as  the  endpoints
+                    of  a  line). If you hold down the shift key,
+                    Select extends the selection: it selects  the
+                    unselected graphic (or unselects the selected
+                    graphic) you clicked on but does not unselect
+                    other  selections.   Clicking  anywhere other
+                    than on a graphic unselects  everything;  you
+                    may also drag a rubberband rectangle around a
+                    group of graphics to select all  of  them  at
+                    once.    Shortcut:  the  right  mouse  button
+                    invokes Select while  the  mouse  is  in  the
+                    drawing area.
+
+     Move           Move  graphics  from  one  spot  to  another.
+                    Shortcut:  the  middle  mouse  button invokes
+                    Move while the mouse is in the drawing area.
+
+     Scale          Scale graphics about their centers.
+
+     Stretch        Stretch graphics vertically  or  horizontally
+                    while tying down the opposite edge.
+
+     Rotate         Rotate graphics about their centers according
```

Comment:
does this work as a table?

Reply:
I have no idea... How would I figure that out?